src/ui/components/Navigation/Core/FavoritesNameWithAliasItem/FavoritesNameWithAliasItem.tsx

```diff
@@ -41,16 +44,23 @@ export const FavoritesNameWithAliasItem = (props: FavoritesNameWithAliasItemProp
                 <span>{text}</span>
                 {isLocked ? <Icon data={Lock} className={b('lock')} /> : null}
             </div>
-            {isAliasVisible && (
-                <Popover placement={['right', 'left', 'bottom', 'top']} content={name}>
-                    <div
-                        className={b('row-btn', b('edit-alias-btn', {visible: isAliasVisible}))}
-                        onClick={onLabelClick}
+
+            <div className={className} onClick={onLabelClick}>
+                {isAliasVisible ? (
+                    <Popover
+                        placement={['right', 'left', 'bottom', 'top']}
+                        content={
+                            <>
```

Comment:
`<>` -> `<React.Fragment>`

Reply:
done, but why ? <> looks shorter and no need for setting attributes